Index: Source/SRRecorderCell.m
===================================================================
--- Source/SRRecorderCell.m	(리비전 51)
+++ Source/SRRecorderCell.m	(작업 사본)
@@ -13,7 +13,6 @@
 
 #import "SRRecorderCell.h"
 #import "SRRecorderControl.h"
-#import "SRKeyCodeTransformer.h"
 #import "SRValidator.h"
 
 @interface SRRecorderCell (Private)
Index: Source/SRKeyCodeTransformer.m
===================================================================
--- Source/SRKeyCodeTransformer.m	(리비전 51)
+++ Source/SRKeyCodeTransformer.m	(작업 사본)
@@ -117,7 +117,7 @@
     return [NSString class];
 }
 
-
+/*
 //---------------------------------------------------------- 
 //  init
 //---------------------------------------------------------- 
@@ -138,7 +138,7 @@
 	[[NSDistributedNotificationCenter defaultCenter] removeObserver:self];
 	[super dealloc];
 }
-
+*/
 //---------------------------------------------------------- 
 //  reloadCache
 //---------------------------------------------------------- 
